### DISCUSSION TITLE
Assert the displayed document in webdriverIO

### DISCUSSION THREAD

**Discussion by Jegadeeshitera2** [webdriverio/webdriverio#11610] [Community]
How to assert the displayed document in webdriverIO

**Comment by christian-bromann** [comment]
You can use:

- https://webdriver.io/docs/api/expect-webdriverio#tohaveurl
- https://webdriver.io/docs/api/expect-webdriverio#tohavetitle

or assert an element on the page. I am not sure what are you exactly looking for but there are plenty ways to assert a website.
